src/rpc/simulate/compressor/LrCompressor.h

```diff
@@ -126,9 +126,10 @@ namespace Rpc
       IntVec<D> kMeshDimensions_;
 
       /**
-      * IntraCorrelation.
+      * IntraCorrelation in fourier space 
+      * Calculated by IntraCorrlation class.
       */
-      RField<D> intraCorrelation_;
+      RField<D> intraCorrelationK_;
```

Comment:
Please make sure that data variables with names that end in underscores are private or (when absolutely necessary) protected. It's hard for me to tell if this is the case here because I don't see the whole file, but it looks like perhaps it is not